packages/tasit-account/package.json

```diff
@@ -35,5 +35,6 @@
     "@babel/register": "^7.0.0",
     "chai": "^4.2.0",
     "mocha": "^5.2.0"
-  }
+  },
+  "gitHead": "e02f70a8c70721039e2c88d7b55d89f50c72e028"
```

Comment:
Hmm, it's interesting that this ended up here

Reply:
Or is this just a new feature of npm I didn't know about?

Reply:
https://github.com/lerna/lerna/issues/1880#issuecomment-456160357
> gitHead is a temporary property only meant to exist during lerna publish when it is constructing the package tarballs. I don't know why it is appearing in your use case.

Reply:
Good catch. It’s probably from the lerna publish failures. I’ll remove it before we merge this